docs/running/cloud/kubernetes.rst

```diff
@@ -188,7 +188,16 @@ To configure your workflow to run on Kubernetes, you will have to configure seve
 
 #. ``TOIL_KUBERNETES_OWNER`` **should** be set to the username of the user running the Toil workflow. The jobs that Toil creates will include this username, so they can be more easily recognized, and cleaned up by the user if anything happens to the Toil leader. In this example we are using ``demo-user``.
 
-Note that Docker containers cannot be run inside of unprivileged Kubernetes pods (which are themselves containers). The Docker daemon does not (yet) support this. Other tools, such as Singularity in its user-namespace mode, are able to run containers from within containers. If using Singularity to run containerized tools, and you want downloaded container images to persist between Toil jobs, you will also want to set ``TOIL_KUBERNETES_HOST_PATH`` and make sure that Singularity is downloading its containers under the Toil work directory (``/var/lib/toil`` by default) by setting ``SINGULARITY_CACHEDIR``. However, you will need to make sure that no two jobs try to download the same container at the same time; Singularity has no synchronization or locking around its cache, but the cache is also not safe for simultaneous access by multiple Singularity invocations. Some Toil workflows use their own custom workaround logic for this problem; this work is likely to be made part of Toil in a future release.
+Note that Docker containers cannot be run inside of unprivileged Kubernetes pods (which are themselves containers). The Docker daemon does not (yet) support this. Other tools, such as Singularity in its user-namespace mode, are able to run containers from within containers. If using Singularity to run containerized tools, and you want downloaded container images to persist between Toil jobs, some setup may be required:
+
+  On non-Toil managed clusters:
+  You will also want to set ``TOIL_KUBERNETES_HOST_PATH``, and make sure that Singularity is downloading its containers under the Toil work directory (``/var/lib/toil`` by default) by setting ``SINGULARITY_CACHEDIR``.
+
+  On Toil-managed clusters:
+  On clusters created with the ``launch-cluster`` command, no setup is required. ``TOIL_KUBERNETES_HOST_PATH`` is already set to ``/var/lib/toil``. ``SINGULARITY_CACHEDIR`` is set to ``/var/lib/toil/singularity`` which is a shared location; however, you may need to implement locking as shown below or change the location to somewhere else.
```

Comment:
It looks like this comes out as a block quote in RST?
![Screenshot 2024-01-25 at 11 54 22](https://github.com/DataBiosphere/toil/assets/5062495/494a740b-5330-4428-a08f-bd103f61e748)

Maybe these should be underlined subheadings instead. Or just bolded introductory phrases.